src/nxdoc/web-ui/web-ui-tutorials/web-ui-slots.md

```diff
@@ -52,7 +52,7 @@ You can see that `my-element` has its `my-element-property` bound to `aPropertyF
 
 {{#> callout type='info' }}
 - Note that every slot on Web UI exposes a property named `user` by default, which contains the current user object.
-{{/callout}}
+  {{/callout}}
```

Comment:
Looks like the spaces are not necessary here and only add noise to the commit, could be removed for a cleaner PR